hypervisor/init_comm.go

```diff
@@ -79,6 +79,11 @@ func ReadVmMessage(conn *net.UnixConn) (*DecodedMessage, error) {
 		res = append(res, buf[:nr]...)
 		read = read + nr
 
+		// if condition satisified, message length may wrong , read again
+		if nr < want && read < needRead {
+			return ReadVmMessage(conn)
+		}
+
```

Comment:
Because it sometimes will get a impossible long message as follows:
I0426 14:22:34.827787   26408 init_comm.go:91] data length is 582570580
I0426 14:22:34.827794   26408 init_comm.go:72] trying to read 512 bytes
I0426 14:22:34.827804   26408 init_comm.go:72] trying to read 8 bytes
I0426 14:22:34.905799   26408 init_comm.go:87] read 8/8 [length = 0]
I0426 14:22:34.905833   26408 init_comm.go:91] data length is 1462868
I0426 14:22:34.905839   26408 init_comm.go:72] trying to read 512 bytes
I0426 14:22:34.905849   26408 init_comm.go:72] trying to read 8 bytes
Especially when you stop the hyper daemon unproperly ,restart it and launch the first pod.So in order not to be blocked here, I do this to clarify the bad message and start to read a new message.